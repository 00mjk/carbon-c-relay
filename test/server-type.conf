cluster a
	forward
		127.0.0.1:2020 type syslog
		127.0.0.1:2021 type linemode
	;

# this should yield an error
cluster b
	forward
		127.0.0.1:2020
	;
