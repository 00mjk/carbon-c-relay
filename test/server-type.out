server-type.conf:11:1: cannot set type 'linemode' for server 127.0.0.1:2020: server was previously defined with type 'syslog'
	;
	^
failed to read configuration 'server-type.conf'
